doc/install_windows.txt

```diff
@@ -124,8 +124,12 @@ and execute the following from the ``Anaconda Prompt``:
 
 .. code-block:: bash
 
-    $ conda install theano
+    $ conda install -c rdonnelly theano
```

Comment:
We don't want to officially recommend this.  The packages will be in the main repo, but for testing we use this channel.